Cargo.toml

```diff
@@ -34,6 +34,7 @@ description = "Restate makes distributed applications easy!"
 [workspace.dependencies]
 # Own crates
 codederror = { path = "crates/codederror" }
+mock-service-endpoint = { path = "tools/mock-service-endpoint" }
```

Comment:
With this PR, the mock service handler is now usable from other packages - this is handy for e2e testing.